src/components/page/JoinNow.svelte

```diff
@@ -1,22 +1,95 @@
 <script>
   import { onMount } from "svelte";
   import Arrow from "@components/images/Arrow.svelte";
+  import IMask from 'imask';
 
-  let form;
-  let input;
+  let form, formTop;
+  let firstName;
+
+  let joinFormTestGroup;
+  const joinFormTestGroups = {
+    NEW: "New Join Form", // Redirects to https://join.futuresuper.com.au/
+    OLD: "Old Join Form", // Redirects to https://join-now.futuresuper.com.au/
+  };
+
+  let joinFormUrl = "https://join.futuresuper.com.au/";
+  $: joinFormUrl = 
+    joinFormTestGroup == joinFormTestGroups.NEW
+      ? "https://join.futuresuper.com.au/"
+      : "https://join-now.futuresuper.com.au/";
 
   onMount(async () => {
+    form.addEventListener('submit', handleFormSubmit);
+
+    const rand = Math.random();
+    joinFormTestGroup =
+      rand > 0.5 ? joinFormTestGroups.NEW : joinFormTestGroups.OLD;
+
     if (window.innerWidth <= 800) {
       setTimeout(() => {
         window.scrollTo({
           top: form.offsetTop,
           behavior: "smooth",
         });
-        input.setAttribute("autofocus", "autofocus");
-        input.focus();
+        firstName.setAttribute("autofocus", "autofocus");
+        firstName.focus();
       }, 2500);
     }
+
+    analytics.track("Experiment Viewed", {
+      experimentId: 'FUM-188',
+      variationName: joinFormTestGroup,
+      property: 'website',
+    });
   });
+
+  // Hidden input field necessary to make sure the value in mobileInput doesn't change as the form submits
+  let mobileInput, hiddenMobileInput;
+  let mask;
+
+  // Reactive because mobileInput is conditionally rendered
+  $: if (joinFormTestGroup == joinFormTestGroups.NEW && mobileInput) {
+    mask = IMask(mobileInput, {
+      mask: ['0000 000 000', '+61 000 000 000'],
+    });
+
+    mobileInput.addEventListener('input', () => {
+      // Regular expression for Australian mobile numbers
+      const mobileRegex = /^(?:04|4|\+614|\+6104)[0-9]{8}$/;
+
+      if (!mobileRegex.test(mask.unmaskedValue)) {
+        mobileInput.setCustomValidity("Invalid phone number. Please enter a valid Australian phone number.");
+      } else {
+        mobileInput.setCustomValidity("");
+      }
+    });
+  }
+
+  const handleFormSubmit = (event) => {
+    event.preventDefault();
+
+    hiddenMobileInput.value = formattedMobileNumber();
+
+    // Regular expression for the mobile number format the join form accepts
+    const mobileRegex = /^(?:04)[0-9]{8}$/;
+    if (mobileRegex.test(hiddenMobileInput.value)) {
+      form.submit();
+    } else {
+      // We shouldn't hit this case if the input mask and input listener worked as expected
+      alert('Invalid phone number. Please enter a valid Australian phone number.');
```

Comment:
We might actually prefer to let the user just submit here and hope that the user fixes it within the join form. Maybe this check is entirely unnecessary since I can't think of a reason why the user would end up here.